### DISCUSSION TITLE
How to make intelij exits code complete window directly to navigation mode while typing?

### DISCUSSION THREAD

**Discussion by urosjarc** [JetBrains/ideavim#307] [Q&A]
Hi I have question about code completing window.
When I type the code complete window opens that shows hints and if i press escape I want my ideavim to jump straight to navigation mode but instead it joust stays in insert mode and only completion window closes... I never want to stay in insert mode and I have enough double pressing my escape key :(

Here is an example... On third line i open code complete window and then escape, but then I must press escape again to go to normal navigation mode...

![Peek 2021-05-07 11-38](https://user-images.githubusercontent.com/8678755/117430982-f80baf00-af28-11eb-87f8-e3a5c90b248c.gif)

I even try to record a macro in which I mapped Esc to triple Esc...

![Peek 2021-05-07 15-33](https://user-images.githubusercontent.com/8678755/117460925-5185d500-af4d-11eb-82d0-5b46a67db62d.gif)

**Comment by AlexPl292** [comment]
Hi! We can't change the current behavior, but I think we can provide an option to configure this behavior.
Sorry, I can't find a corresponding issue on YouTrack, so I'll just notify you when this feature will be available.

**Reply by urosjarc** [reply]
Yes thank you wery much!

**Reply by crusj** [reply]
now is available?
